docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.4"
+
+services:
+  server:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    volumes:
+      - ./backend/:/backend/
+    entrypoint: ./docker-start.sh
+    container_name: taia-x-backend
+    env_file:
+      - ./backend/.env
+    ports:
+      - 5678:5678
+      - 8000:8000
+    depends_on:
+      - db
+      - ipfs
+
+  frontend:
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile
+    volumes:
+      - ~/.composer-docker/cache:/root/.composer/cache:delegated
```

Comment:
@amrap030, just for me to understand, what does this command do? does this command enable the hot-loading for the image?

Reply:
No, it doesn't enable hot-reloading. I was looking for how to boost the performance of using webpack inside docker and this was suggested. But to be honest, I can't notice any difference, so this could also be removed. After a quick research, I found that this statement is good to re-use local composer cache via host-volume.